PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -1006,6 +1008,7 @@ void AliAnalysisTaskGammaConvV1::UserCreateOutputObjects(){
   }
   fHistoNGoodESDTracks      = new TH1F*[fnCuts];
   fHistoVertexZ             = new TH1F*[fnCuts];
+  fHistoEventSphericity     = new TH1F*[fnCuts];
```

Comment:
please encapsulate this in the " if(((AliConvEventCuts*)fEventCutArray->At(iCut))->GetUseSphericity()!=0){" condition